torch/csrc/jit/export.cpp

```diff
@@ -1317,6 +1318,22 @@ std::string pretty_print_onnx(
   return prettyPrint(graph_encoder.get_model_proto());
 }
 
+void protobuf_log_handler(
+    ::google::protobuf::LogLevel level,
+    const char* filename,
+    int line,
+    const std::string& message) {
+  if (level == ::google::protobuf::LogLevel::LOGLEVEL_INFO) {
+    LOG(INFO) << message;
+  } else if (level == ::google::protobuf::LogLevel::LOGLEVEL_WARNING) {
+    LOG(WARNING) << message;
+  } else if (level == ::google::protobuf::LogLevel::LOGLEVEL_ERROR) {
+    LOG(ERROR) << message;
```

Comment:
Currently, this logger handler only redirect log message to console,
```
[E ..\torch\csrc\jit\export.cpp:1331] Exceeded maximum protobuf size of 2GB: 2200000396
```
Is there any proper way to propagate this error message to the user as an python exception?